src/test-func/groovy/org/transmartproject/batch/highdim/mrna/platform/MrnaPlatformCleanScenarioTests.groovy

```diff
@@ -155,4 +154,24 @@ class MrnaPlatformCleanScenarioTests {
         )
     }
 
+    @Test
+    void testAnnotationDate() {
+        def q = """
+                SELECT annotation_date organism
+                FROM ${Tables.GPL_INFO}
+                WHERE platform = :platform"""
+        def p = [platform: PLATFORM_ID_NORM]
+
+        Date date = jdbcTemplate.queryForObject(q, p, Date)
+
+        def execution = jobRepository.getLastJobExecution(
+                MrnaPlatformJobConfiguration.JOB_NAME,
+                RUN_JOB_RULE.jobParameters)
+        Date expectedDate = execution.startTime
+
+        assertThat date, allOf(
+                is(notNullValue()),
+                is(expectedDate))
```

Comment:
Do you still need to check wether it's null if you have comparing against concrete value?